docs/core/diagnostics/observability-with-otel.md

```diff
@@ -0,0 +1,429 @@
+---
+title: .NET Observability with OpenTelemetry
+description: An introduction to observing .NET apps with OpenTelemetry
+ms.date: 6/14/2023
+ms.topic: conceptual
+---
+
+# .NET observability with OpenTelemetry
+
+When you run a distributed application, you want to know how well the app is performing and to detect potential problems before they become larger. One way that you can do this is by monitoring specific instrumentation emitted by the application and looking for changes in metrics and logs.
+
+## What is observability
+
+Observability in the context of a distributed system is the ability to monitor and analyze telemetry about the state of each component, to be able to observe changes in performance, and to diagnose why those changes occur. Unlike debugging, which is invasive and can affect the operation of the application, observability is intended to be transparent to the primary operation and have a small enough performance impact that it can be used continuously.
+
+Observability is commonly done using a combination of:
+
+- Logs - record individual operations, such as an incoming request, a failure in a specific component, or an order being placed.
+- Metrics - measuring counters and gauges such as number of completed requests, active requests, widgets that have been sold; or a histogram of the request latency.
```

Comment:
Minor suggestion - I don't know how many folks are familiar with histogram, maybe something like "99% latency" will be easier for them to understand?